modules/widgets/simple-payments.php

```diff
@@ -0,0 +1,330 @@
+							</button>
+						</label>
+					</li>
+				<?php endforeach; ?>
+			</ul>
+		</div>
+		<?php
+	}
+
+	protected function render_product_form( $product_id, $hide ) {
+		$product_args = $this->get_product_args( $product_id );
+		$price = ( $product_args['price'] ) ? esc_attr( Jetpack_Money_Format::format_price_amount( $product_args['currency'], $product_args['price'] ) ) : '';
+		?>
+		<div class="simple-payments-form" <?php if ( $hide ) echo 'style="display:none;"'; ?>>
+			<p>
+				<label for="<?php esc_attr_e( $this->get_field_id( 'name' ) ); ?>"><?php esc_html_e( 'What are you selling?', 'jetpack' ); ?></label>
+				<input <?php echo empty( $products ) ? '' : 'disabled'; ?> class="widefat field-name" id="<?php esc_attr_e( $this->get_field_id( 'name' ) ); ?>" name="<?php esc_attr_e( $this->get_field_name( 'name' ) ); ?>" type="text" placeholder="<?php esc_attr_e( 'Product name', 'jetpack' ); ?>" value="<?php esc_attr_e( $product_args['name'] ); ?>" />
+			</p>
+			<div class="simple-payments-image-fieldset">
+				<label><?php esc_html_e( 'Product image', 'jetpack' ); ?></label>
+				<div class="placeholder" <?php if ( has_post_thumbnail( $product_id ) ) echo 'style="display:none;"'; ?>><?php esc_html_e( 'Select an image', 'jetpack' ); ?></div> <!-- TODO: actual placeholder -->
+				<div class="simple-payments-image" data-image-field="<?php esc_attr_e( $this->get_field_name( 'image' ) ); ?>"> <!-- TODO: hide if empty, CSS? -->
+					<?php
+					if ( has_post_thumbnail( $product_id ) ) {
+						$image_id = get_post_thumbnail_id( $product_id );
+						echo '<img src="' . esc_url( wp_get_attachment_image_url( $image_id, 'full' ) ) . '" />';
+						echo '<input type="hidden" name="' . $this->get_field_name( 'image' ) . '" value="' . esc_attr( $image_id ) . '" />';
+					}
+					?>
+					<button class="button simple-payments-remove-image"><span class="screen-reader-text"><?php esc_html_e( 'Remove image', 'jetpack' ); ?></span></button>
+				</div>
+			</div>
+			<p>
+				<label for="<?php esc_attr_e( $this->get_field_id( 'description' ) ); ?>"><?php esc_html_e( 'Description', 'jetpack' ); ?></label>
+				<textarea class="field-description widefat" rows=5 id="<?php esc_attr_e( $this->get_field_id( 'description' ) ); ?>" name="<?php esc_attr_e( $this->get_field_name( 'description' ) ); ?>"><?php  esc_html_e( $product_args['description'] ); ?></textarea>
+			</p>
+			<p class="cost">
+				<label for="<?php esc_attr_e( $this->get_field_id( 'price' ) ); ?>"><?php esc_html_e( 'Price', 'jetpack' ); ?></label>
+				<select class="field-currency widefat" id="<?php esc_attr_e( $this->get_field_id( 'currency' ) ); ?>" name="<?php esc_attr_e( $this->get_field_name( 'currency' ) ); ?>">
+					<?php foreach( Jetpack_Money_Format::get_currencies_map() as $code => $currency ) { ?>
+						<option value="<?php esc_attr_e( $code ) ?>"<?php selected( $product_args['currency'], $code ); ?>>
```

Comment:
Should this be `echo esc_attr` instead of `esc_attr_e` here, since it is not translatable here, we do not have a textdomain?